src/widgets/query-rule-custom-data/query-rule-custom-data.tsx

```diff
@@ -54,7 +56,11 @@ const renderer: Renderer<
   const { container, cssClasses, templates } = widgetParams;
 
   render(
-    <CustomData cssClasses={cssClasses} templates={templates} items={items} />,
+    <CustomData
+      cssClasses={cssClasses as QueryRuleCustomDataComponentCSSClasses}
```

Comment:
unlike the others, it seems that this component doesn't use cx on the class names, causing you to cast, that's probably wrong?

Reply:
Does this seems better? https://github.com/algolia/instantsearch.js/pull/4774/commits/18d2b5ffb91771aecc5905a46d632306769383b4

Reply:
yes, slightly bigger, but consistent now